packages/preferences-controller/src/PreferencesController.ts

```diff
@@ -132,6 +132,10 @@ export type PreferencesState = {
    * Controls whether balance and assets are hidden or not
    */
   privacyMode: boolean;
+  /**
+   * Allow user to stop being prompted for smart account upgrade
+   */
+  dismissSmartAccountSuggestionEnabled: boolean;
```

Comment:
nit: in the past we tried to avoid using Hungarian notations for flags here; do you think we can adjust the wording here as well? 
```suggestion
  dismissSmartAccountSuggestion: boolean;
```